node-fs-writefile/note-to-self.js

```diff
@@ -0,0 +1,16 @@
+const fs = require('fs');
+
+const note = process.argv;
```

Comment:
So, you've made your life more difficult...

Typically, in a CLI process, you would pass a single argument to the function by wrapping it in quotes. Then you could extract the entire argument using just `process.argv[2]`.

You typed:
`node note This is my note`
but if you had instead typed:
`note note "This is my note"`
then you could have just used `argv[2]`